wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1018,30 +1019,68 @@ instance BuildableSafeGen PasswordUpdate where
         pwdOld
         pwdNew
 
+data Accuracy
+    = Accurate
+    | LowerBound
```

Comment:
What about some comments for the lost soul trying to find purpose into this type? 😀 I know it's self descriptive, but let's try to be good open source citizens 😁